cookbooks/bcpc-hadoop/attributes/hive.rb

```diff
@@ -1,3 +1,5 @@
-default[:bach][:hive][:metastore][:port] = 9083
-default[:bach][:hive][:hiveserver2][:port] = 10000
-default[:bcpc][:hive][:heap][:size]=1024
+default[:bcpc][:hadoop][:hive][:heap][:size]=1024
+default[:bcpc][:hadoop][:hive][:hiveserver2][:authentication][:method] = "KERBEROS"
+default[:bcpc][:hadoop][:hive][:hiveserver2][:authentication][:ldap_url] = ""
+default[:bcpc][:hadoop][:hive][:hiveserver2][:authentication][:ldap_domain] = ""
```

Comment:
Currently we do not setup `LDAP` for VM cluster. This PR just provides configuration and `LDAP` implementation is up to the user of this configuration.